textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/neextrator/NamedEntityExtractor.java

```diff
@@ -72,7 +84,146 @@ public void open() throws Exception {
      */
     @Override
     public ITuple getNextTuple() throws Exception {
-        return null;
+        if (sourceTuple == null) {
+            return null;
+        } else {
+            List<Span> spanList = new ArrayList<>();
+            for (Attribute attribute : searchInAttributes) {
+                String fieldName = attribute.getFieldName();
+                IField field = sourceTuple.getField(fieldName);
+                spanList.addAll(getSpans(field, fieldName));
+            }
+            IField spanField = new ListField<Span>(spanList);
+            List<IField> fields = new ArrayList<IField>();
+            fields.add(spanField);
+            ITuple resultTuple = new DataTuple(new Schema(SchemaConstants.SPAN_LIST_ATTRIBUTE), fields.toArray(new IField[fields.size()]));
```

Comment:
@chenlica .. We don't have any helper function for this. It is a normal constructor call.